modules/devshells.nix

```diff
@@ -12,7 +12,6 @@
     devShells = {
       default = pkgs.mkShell {
         buildInputs = with pkgs; [
-          rnix-lsp
```

Comment:
Can you replace with `nil`?

Reply:
https://github.com/oxalica/nil

Reply:
As it is part of `nixpkgs` it seems that simply adding `nil` worked, 1949823662f8abe12c41baa694d714a2fb8a0849.
If I run `nix develop #cosmos-shell` and then `nil --version` I get: `nil 2023-08-09`